submissions/LofeWilter/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="nav">
+            <div class="nav_item">Mail</div>
+            <div class="nav_item">Images</div>
+            <div class="nav_item">
+                <label for="moreservices" tabindex="0" class="test">
+                    <img src="assets/popup-button.png" alt="">
+                </label>
+            </div>
+
+            <div class="nav_item">
+                <img src="assets/bell.png" alt="">
+            </div>
+            <div class="nav_item">
+                <img src="assets/favicon.png" alt="">
+            </div>
+        </div>
+        <input type="checkbox" id="moreservices">
+        <div class="wrapper">
+            <div class="services">
+                <a tabindex="0">
```

Comment:
You should set the `href` attribute and maybe you don't need the `tabindex` attribute here